numba/core/typing/builtins.py

```diff
@@ -449,7 +445,16 @@ def generic(self, args, kws):
 
 
 @infer_global(operator.ne)
-class ConstOpNotEq(ConstOpEq):
+class ConstOpNotEq(AbstractTemplate):
+    def generic(self, args, kws):
+        assert not kws
+        (arg1, arg2) = args
+        if isinstance(arg1, types.Literal) and isinstance(arg2, types.Literal):
+            return signature(types.boolean, arg1, arg2)
```

Comment:
Why doesn't subclassing `ConstOpEq` as was done before work? Seems silly to repeat this function body at any rate.

Reply:
@eric-wieser Yep, thanks. I'll revert this change.